swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -6870,16 +7011,32 @@ components:
             $ref: "#/components/schemas/paymentInitiationBulkElement_json"
         transactionStatus:
           $ref: "#/components/schemas/transactionStatus"
+        tppMessage:
+          type: array
+          items: 
+            $ref: "#/components/schemas/tppMessageGeneric"
+          description: Messages to the TPP on operational issues.
 
 
     scaStatusResponse:
       description: Body of the JSON response with SCA Status.
+      required: 
+        - scaStatus
       type: object
       properties:
         scaStatus:
           $ref: "#/components/schemas/scaStatus"
+        psuMessage:
+          $ref: "#/components/schemas/psuMessageText"
```

Comment:
Relevant Change: psuMessage is new. We already have it
Adoption approach: Nothing to change